migration_original/ODS1Stage/tables/Mid/Practice/spu_translated_Practice.sql

```diff
@@ -0,0 +1,511 @@
+
+-- If statement two possible pracice batch tables
+
+/*
+WITH CTE_PracticeBatch AS (
+    SELECT PracticeID
+    FROM Base.Practice
+    ORDER BY PracticeID
+)
+
+*/
+
+WITH CTE_PracticeBatch AS (
+    SELECT DISTINCT e.PracticeID
+    -- We havent decided where this information is gonna live in snowlfake platform
+    FROM Snowflake.etl.ProviderDeltaProcessing as a -- !!!!!!!!!!!!
```

Comment:
Replace aliases letters from a,b,c to a more descripted letters.
From example ProviderToOffice AS pto